manager/orchestrator/task_reaper.go

```diff
@@ -75,13 +76,18 @@ func (tr *TaskReaper) Run() {
 				}] = struct{}{}
 				if len(tr.dirty) > maxDirty {
 					tr.tick()
+					timer.Stop()
```

Comment:
Maybe better to put `timer.Stop` before `tr.tick()`, in case `tr.tick()` runs long.